ui/app/AppLayouts/Chat/views/ChatMessagesView.qml

```diff
@@ -213,7 +213,6 @@ Item {
             id: msgDelegate
 
             width: ListView.view.width
-            height: implicitHeight
```

Comment:
Why is this removed?

Reply:
Binding loop + it causes to bug when you create a private group chat and your first initial message is cut (not fully visible). Adding emoji/edit/etc makes msg display ok

Reply:
Curious; we had this back and forth before; what Qt version are you using? I don't see any binding loop warning at all (Qt 5.15.8 here)

Reply:
Qt 5.15.2 (macos)

Reply:
Ok.
With setting height:

https://user-images.githubusercontent.com/117639195/225005755-44e207b2-0c5a-4c54-910e-292d7e5a6658.mov

Without setting height:

https://user-images.githubusercontent.com/117639195/225005893-bc923f5d-6da6-4baf-bf88-c013a320615f.mov